cishouseholds/validate_class.py

```diff
@@ -26,6 +26,10 @@ def __init__(self, dataframe: DataFrame, error_column_name: str) -> None:
             "between": {"function": self.between, "error_message": "{} should be between {}{} and {}{}"},
             "null": {"function": self.not_null, "error_message": "{} should not be null"},
             "valid_vaccination": {"function": self.valid_vaccination, "error_message": "invalid vaccination"},
+            "valid_file_date": {
+                "function": self.check_valid_file_date,
+                "error_message": "file date should be after visit date",
```

Comment:
We're checking the visit date in particular, so reverse wording focus to this. Also describe the rest of the check.

Please substitute in the actual column names using `{}`

```suggestion
                "error_message": "visit date should be before file date when samples have been collected",
```